DQM/L1TMonitor/python/L1TStage2uGT_cfi.py

```diff
@@ -1,10 +1,11 @@
 import FWCore.ParameterSet.Config as cms
-
 from DQMServices.Core.DQMEDAnalyzer import DQMEDAnalyzer
 l1tStage2uGT = DQMEDAnalyzer('L1TStage2uGT',
     l1tStage2uGtSource = cms.InputTag("gtStage2Digis"),    
     monitorDir = cms.untracked.string("L1T/L1TStage2uGT"),
     verbose = cms.untracked.bool(False),
     firstBXInTrainAlgo = cms.untracked.string("L1_FirstCollisionInTrain"),
-    lastBXInTrainAlgo = cms.untracked.string("L1_LastCollisionInTrain")
+    lastBXInTrainAlgo = cms.untracked.string("L1_LastCollisionInTrain"),    
+    unprescaledAlgoShortList = cms.untracked.vstring("L1_SingleMu25","L1_SingleEG32","L1_DoubleMu_15_7","L1_SingleJet200","L1_HTT450er"),
+    prescaledAlgoShortList = cms.untracked.vstring("L1_SingleMu3","L1_SingleEG15","L1_SingleJet90","L1_DoubleJet40er3p0","L1_HTT200er") 
```

Comment:
Please replace the list of algos with this one:
`prescaledAlgoShortList = cms.untracked.vstring("L1_SingleMu0_BMTF”, “L1_SingleMu0_OMTF”, “L1_SingleMu0_EMTF”, “L1_SingleEG10er2p5”, ”L1_SingleEG26er2p5”, “L1_SingleLooseIsoEG28er1p5”, ”L1_SingleJet60”, “L1_SingleJet60er2p5”, “L1_SingleJet60_FWD3p0”, “L1_ETMHF100”, ”L1_HTT120er”, “L1_ETT1200”)`